### PR TITLE
Surgical Apron can now hold more things.

### PR DESCRIPTION
**Pull request by OHJ4Y**
## About The Pull Request

Exactly what it says on the tin. This PR makes surgical aprons hold more things.

## Why It's Good For The Game

Presently, surgical aprons only hold surgery tools, which any medical doctor would store in their medical belt/medkit. This PR aims to make surgical aprons a more viable (and fashionable) choice for doctors looking at their wardrobes.

## Changelog
:cl:
qol: Surgical aprons can now hold more medical-related items in their storage slot.
/:cl:

### CODE REVIEW

File: code/modules/clothing/suits/jobs.dm [hunk 1]
```diff
@@ -155,7 +155,18 @@
 	name = "surgical apron"
 	desc = "A sterile blue surgical apron."
 	icon_state = "surgical"
-	allowed = list(/obj/item/scalpel, /obj/item/surgical_drapes, /obj/item/cautery, /obj/item/hemostat, /obj/item/retractor)
+	allowed = list(
+		/obj/item/analyzer,
```
Comment (GuillaumePrata) [inline code/modules/clothing/suits/jobs.dm:159]:
```suggestion
		/obj/item/healthanalyzer,
```
This analyzer there is the Gas/Atmos one and not the health analyzer.

Comment (OHJ4Y) [reply code/modules/clothing/suits/jobs.dm]: Fixed.

File: code/modules/clothing/suits/jobs.dm [hunk 2]
```diff
@@ -155,7 +155,17 @@
 	name = "surgical apron"
 	desc = "A sterile blue surgical apron."
 	icon_state = "surgical"
-	allowed = list(/obj/item/scalpel, /obj/item/surgical_drapes, /obj/item/cautery, /obj/item/hemostat, /obj/item/retractor)
```
Comment (ATH1909) [inline code/modules/clothing/suits/jobs.dm:158]: would you mind readding these items (and maybe some of the other surgery tools) to the allowed list of surgical aprons? idk, maybe some doctor out there likes keeping their (laser) scalpel in their suit storage slot or something.

Comment (OHJ4Y) [reply code/modules/clothing/suits/jobs.dm:158]: Re-added surgical tools to the list, added advanced tools as well

**Comment by Fikou** [comment]
changelogs dont work this way

File: code/modules/clothing/suits/jobs.dm [hunk 3]
```diff
@@ -155,7 +155,25 @@
...
+	allowed = list(
+		/obj/item/hemostat,
+		/obj/item/cautery,
+		/obj/item/scalpel,
+		/obj/item/surgical_drapes,
+		/obj/item/retractor,
+		/obj/item/bonesetter,
+		/obj/item/scalpel/advanced,
+		/obj/item/retractor/advanced,
```
Comment (coiax) [inline code/modules/clothing/suits/jobs.dm:166]: Aren't these redundant? I assume that the allowed suit slot checks for subtypes.

**Review by coiax — CHANGES_REQUESTED** [review]

File: code/modules/clothing/suits/jobs.dm (hunk 3 above)
Comment (OHJ4Y) [reply code/modules/clothing/suits/jobs.dm]: apologies for the late reply i was doing... stuff.

tested locally, those two lines were indeed redundant. Removed.

**Comment by OHJ4Y** [comment]
Hell yea

**Pull request merged**